### DISCUSSION TITLE
How do I trigger to run an animation and hide it after it ends?

### DISCUSSION THREAD

**Discussion by colossus21** [yohamta/ganim8#20] [General]
I am working on a game project which would require special effect sort of thing. I want to hide the animation just after it runs once when a button is triggered.

**Comment by yohamta0** [comment]
Thanks for the question. Currently there’s no way to render nothing after an anime ends. One possible way would be removing the instance from your game system if the frame position is at the end of the animation.

maybe it's also possible to add a method or options that allows us to do it easier.

**Reply by colossus21** [reply]
Thanks Yohamta.  I would be glad to add a method sooner.
